### ISSUE TITLE
cuelang.org: preprocessor: allow file-level #nofmt markers inside {{{with code}}} blocks

### ISSUE THREAD

**Issue by jpluscplusm** [cue-lang/cue#2739]
**Is your feature request related to a problem? Please describe.**

Alpha contains some markdown files where only a subset of the files in a txtar `{{{with code}}}` block need to be left formatted as given, and not passed through the filetype-specific formatter.

Currently, whilst comments in a preprocessor test file (https://github.com/cue-lang/cuelang.org/blob/244206ce20da58c45b06da2ff2334ffe25bae136/internal/cmd/preprocessor/cmd/testdata/execute_code_tags.txtar) suggest that `#nofmt` accepts arguments, neither the implementation nor CI suggest that these arguments have any effect. The entire txtar node is treated as `#nofmt`, regardless of any parameters present. I'm filing this issue as a feature request, as I don't see the promise of `#nofmt` accepting parameters documented anywhere firmly. 

As an example, here's a CL where I reformatted some comments: https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1173843. In the resulting CI deployment (https://cl-1173843-2--cue-cls.netlify.app/docs/concept/data-validation-use-case/#client-side-validation) notice how the comment at the end of the first line of `check.cue` has too much space before it, and thus must not have been formatted by the preprocessor (or CI would have failed). This contrasts with the source (line 33 of https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1173843/2/content/docs/concept/data-validation-use-case/en.md), where `#nofmt` is given a parameter of `ranges.yaml`.

**Describe the solution you'd like**

I'd like `#nofmt` to accept arguments dictating which files don't need formatting, and for every /other/ file present in the txtar to be formatted.
 
**Describe alternatives you've considered**

Manual formatting of the other files in the txtar.

**Additional context**

n/a

**Comment by myitcv** [comment]
Does this work?

https://github.com/cue-lang/cuelang.org/blob/244206ce20da58c45b06da2ff2334ffe25bae136/internal/cmd/preprocessor/cmd/testdata/execute_multistagescript.txtar#L74

**Comment by jpluscplusm** [comment]
> Does this work?

No; if it worked inside `{{{with code ...}}}` blocks then this trybot would have failed: https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1176139
